Dockerfile

```diff
@@ -29,8 +29,8 @@ RUN mkdir /home/alfio/app
 WORKDIR /home/alfio/app
 
 RUN mkdir logs
-COPY --chown=alfio WEB-INF WEB-INF
-COPY --chown=alfio resources resources
+COPY --chown=alfio src/main/webapp/WEB-INF    WEB-INF
```

Comment:
I see. Then this approach wouldn't really work. Would you be open to adding build steps inside the `Dockerfile`? This way it would be more reproducible too.

Reply:
maybe we can import the same content from build/dockerize/ ?
Prerequisite: we have to run ./gradlew distribution before trying to copy the files

Reply:
+1 to have build steps in the Dockerfile. That would be great :)

Reply:
but then you should [use multi-stage builds](https://docs.docker.com/develop/develop-images/multistage-build/) ... 

PS, just FYI: My #524 does something similar (using S2I instead of Dockerfile) - we can both, of course.

Reply:
@vorburger what do you mean by that? AFAIK, the Dockerfile is already a multi-stage build, see https://github.com/alfio-event/alf.io/blob/master/src/main/dist/Dockerfile#L15:L18

Reply:
@cbellone oups sorry I hadn't seen that, not sure what I was looking at.. ignore!